src/components/shared/Layout.tsx

```diff
@@ -15,6 +16,7 @@ export interface SceneProps {
   expo: Exposition[];
   image: string;
   imageAlt?: string;
+  style?: CSS.Properties;
```

Comment:
Does this work?

```suggestion
  style?: React.CSSProperties;
```

Reply:
it says React has no exported member CSS

Reply:
but it still works, so ig its good